exercises/sgf-parsing/SgfParsingTest.cs

```diff
@@ -31,7 +31,7 @@ public void Node_without_tree()
     public void Node_without_properties()
     {
         var encoded = "(;)";
-        var expected = new SgfTree(new Dictionary<string, string[]>());
+        var expected = new SgfTree(new Dictionary<string, string[]>() );
```

Comment:
Super minor but do you know why the generator is generating these extra spaces here (they're in the rest of the file as well).

Reply:
I found the issue. PR has been updated.

And I appreciate this minor nits by the way :)